var/spack/repos/builtin/packages/hipblas/package.py

```diff
@@ -69,3 +69,8 @@ def cmake_args(self):
 
     def setup_build_environment(self, env):
         env.set('CXX', self.spec['hip'].hipcc)
+
+    @property
+    def headers(self):
+        return find_headers('hipblas', self.prefix.include, recursive=False) \
+            or None
```

Comment:
Two questions:

* Do you specifically search for `hipblas` to avoid pulling in headers (the default implementation grabs all `.h` files)?
* Would you expect that this would ever not find headers? If it should always find headers, you can `raise NoHeaderError` (the same behavior as the default `.headers` implementation)